samples/compose-samples/src/main/java/com/squareup/sample/compose/hellocomposeworkflow/HelloComposeWorkflow.kt

```diff
@@ -57,5 +59,6 @@ fun HelloComposeWorkflowPreview() {
     onOutput = {},
     runtimeConfig = AndroidRuntimeConfigTools.getAppWorkflowRuntimeConfig()
   )
-  WorkflowRendering(rendering, ViewEnvironment.EMPTY)
+  // not sure if I should be adding the factory here
```

Comment:
Yeah, I think this is the problem. Looking for the right place…

Reply:
You were so close. Put this on line 57:

```kotlin
  val rendering by HelloComposeWorkflow
    .mapRendering {
      it.withEnvironment(
        ViewEnvironment.EMPTY.plus(Pair(ComposableFactoryKey, HelloFactory))
      )
    }
```

Reply:
Oops, nope. That _should_ have fixed it, but we need to add support to `VisualFactory` support for `EnvironmentScreen`.

Reply:
Haven't done that yet on the classic view spike either, instead I make `AndroidScreen` work and was careful to use that on anything I've tried to render so far.

Reply:
You can see in `AndroidViewMultiRendering` how I added support for `Named` and `NamedScreen` for classic views, along with a lot of hand wringing about whether I did it in the right place or not.